### PR TITLE
[AAE-7855] variables in process definition

### PR DESCRIPTION
**Pull request by jsokolowskii**
This PR is adding query param `include`, which now only takes `variables` value. If this param is in the request, the endpoint returns displayable variable definitions for each process definition.

### CODE REVIEW

File: activiti-cloud-api/activiti-cloud-api-process-model/src/main/java/org/activiti/cloud/api/process/model/CloudProcessDefinition.java [hunk 1]
```diff
@@ -16,8 +16,16 @@
...
 
 import org.activiti.api.process.model.ProcessDefinition;
+import org.activiti.api.process.model.VariableDefinition;
 import org.activiti.cloud.api.model.shared.CloudRuntimeEntity;
 
+import java.util.List;
+
 public interface CloudProcessDefinition extends CloudRuntimeEntity, ProcessDefinition {
 
+    @Override
```
Comment (jesty) [inline activiti-cloud-api/activiti-cloud-api-process-model/src/main/java/org/activiti/cloud/api/process/model/CloudProcessDefinition.java]: Why do we need it?

Comment (jsokolowskii) [reply activiti-cloud-api/activiti-cloud-api-process-model/src/main/java/org/activiti/cloud/api/process/model/CloudProcessDefinition.java]: I've added `getVariableDefinitions()` to ProcessDefinition in activiti core, but in activiti-cloud ProcessInstanceEntity implements it (by implementing CloudProcessDefinition). This way I won't have to implement this method in entity, which would look weird.

Comment (jsokolowskii) [reply activiti-cloud-api/activiti-cloud-api-process-model/src/main/java/org/activiti/cloud/api/process/model/CloudProcessDefinition.java]: The implementation moved to ExtendedCloudProcessDefinition.

**Review by jesty — APPROVED** [review]
LGTM

**Review by miguelruizdev — APPROVED** [review]

**Review by jesty — CHANGES_REQUESTED** [review]
I would just check a doubt about the new dependency

**Review by jesty — APPROVED** [review]
done, LGTM

**Pull request merged**